components/file_attachment/file_attachment.tsx

```diff
@@ -121,6 +139,124 @@ export default class FileAttachment extends React.PureComponent<Props, State> {
         }
     }
 
+    refCallback = (menuRef: Menu) => {
+        if (menuRef) {
+            const anchorRect = this.buttonRef.current?.getBoundingClientRect();
+            let y;
+            if (typeof anchorRect?.y === 'undefined') {
+                y = typeof anchorRect?.top == 'undefined' ? 0 : anchorRect?.top;
```

Comment:
Shouldn't we use a triple equals?

```suggestion
                y = typeof anchorRect?.top === 'undefined' ? 0 : anchorRect?.top;
```